### DISCUSSION TITLE
CF-812AC is working.

### DISCUSSION THREAD

**Discussion by duyatmb** [chris1111/Wireless-USB-OC-Big-Sur-Adapter#73] [General]
I have a "Comfast 812-AC" (CF-812AC) and it's working on BigSur 11.1.

**Comment by chris1111** [comment]
Thanks I will add in the list

**Comment by andersomdutra** [comment]
> I have a "Comfast 812-AC" (CF-812AC) and it's working on BigSur 11.1.

Can you help me?
I have CF-812AC V2 and I would like to know if I needed to do anything different to make it work.
Which kexts did you use too.

_My version of macos is Ventura._

**Reply by andersomdutra** [reply]
I returned the CF-812AC V2.
I bought the CF-812AC just like @duyatmb and it worked too, as expected.
